test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -70,6 +70,62 @@ public void TensorFlowTransformMatrixMultiplicationTest()
             }
         }
 
+        private class TestDataInt
+        {
+            [VectorType(4)]
+            public long[] a;
+            [VectorType(4)]
+            public int[] b;
+        }
+
+        /// <summary>
+        /// Test to ensure the support for int and int64 types.
+        /// </summary>
+        [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))] // TensorFlow is 64-bit only
+        public void TensorFlowTransformAdditionOfIntAndInt64Test()
```

Comment:
>TensorFlowTransformAdditionOfIntAndInt64Test [](start = 20, length = 44)

Hi Zeeshan, is there supposed to be a test for the 32-bit model? I thought the other tests were 32-bit models, but It looks like the other tests either are skipped or disable unless its a 64-bit process.

Reply:
TensorFlow works only on 64-bit platform. So, i think none of the TF tests are 32-bit.

---
In reply to: [246982838](https://github.com/dotnet/machinelearning/pull/2101#discussion_r246982838) [](ancestors = 246982838)

Reply:
ah! sorry I misread, you are handling int64 and int32 types...so then do the other tests use models that are using int32s?


---
In reply to: [246982838](https://github.com/dotnet/machinelearning/pull/2101#discussion_r246982838) [](ancestors = 246982838)

Reply:
actually I see -- you have a TestDataInt class that has both long and int....

---
In reply to: [246983378](https://github.com/dotnet/machinelearning/pull/2101#discussion_r246983378) [](ancestors = 246983378,246982838)